aspnetcore/host-and-deploy/visual-studio-publish-profiles.md

```diff
@@ -438,17 +447,18 @@ MSBuild file.
     <DeleteExistingFiles>False</DeleteExistingFiles>
   </PropertyGroup>
   <ItemGroup>
-    <ResolvedFileToPublish Include="..\ReadMe2.MD">
-      <RelativePath>wwwroot\ReadMe2.MD</RelativePath>
+    <ResolvedFileToPublish Include="..\ReadMe2.md">
+      <RelativePath>wwwroot\ReadMe2.md</RelativePath>
     </ResolvedFileToPublish>
 
     <Content Update="wwwroot\Content\**\*" CopyToPublishDirectory="Never" />
     <Content Update="Views\Home\About2.cshtml" CopyToPublishDirectory="Never" />
-
   </ItemGroup>
 </Project>
 ```
 
+The preceding example uses the `ResolvedFileToPublish` item, which is provided by a publish targets file in the .NET Core SDK. The item's definition contains a `<CopyToPublishDirectory>` element whose inner text is `Always`. No `<CopyToPublishDirectory>` override is provided in the markup because file inclusion in the publish directory is the intent.
```

Comment:
```suggestion
The preceding example uses the `<ResolvedFileToPublish>` item, which is provided by a publish targets file in the .NET Core SDK. The item's definition contains a `<CopyToPublishDirectory>` element whose inner text is `Always`. No `<CopyToPublishDirectory>` override is provided in the markup because file inclusion in the publish directory is the intent.
```